scripts/build-images.sh

```diff
@@ -15,28 +15,29 @@
 # limitations under the License.
 
 REPOSITORY=${1}
+
 if [ -z "${TRAVIS_TAG}" ] ; then
     IMAGE_TAG="latest"
 else
     IMAGE_TAG=${TRAVIS_TAG}
 fi
 
 # building and pushing the operator images
-docker build . -t ${REPOSITORY}/eunomia-operator:${IMAGE_TAG} -f build/Dockerfile
-docker push ${REPOSITORY}/eunomia-operator:${IMAGE_TAG}
+docker build . -t ${REPOSITORY}/eunomia-operator:${IMAGE_TAG} -f build/Dockerfile || exit 1
```

Comment:
I think we could change the shebang to `#!/usr/bin/env bash -e` rather than adding `|| exit 1` everywhere